src/Compilers/CSharp/Portable/Syntax/SyntaxEquivalence.cs

```diff
@@ -110,11 +110,13 @@ private static bool AreEquivalentRecursive(GreenNode before, GreenNode after, Fu
 
             if (topLevel)
```

Comment:
The name "topLevel" is somewhat confusing. At least, I wouldn't guess this is about method bodies.